_posts/2024-07-30-low-code-vs-ruby-on-rails-el-éxito-de-cualquier-idea.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: 'Low Code y No Code vs Ruby on Rails: el éxito de cualquier idea'
+author: Santiago Pérez González
+tags: ruby-on-rails no-code low-code programación ideas mvp mvc
+images_path: "/assets/images/2024-07-30-low-code-vs-ruby-on-rails-el-éxito-de-cualquier-idea"
+background: "/assets/images/2024-07-30-low-code-vs-ruby-on-rails-el-éxito-de-cualquier-idea/tech.jpg"
+date: 2024-07-30 08:49 -0500
+---
+En el vertiginoso mundo del desarrollo de aplicaciones, cada idea es un destello de innovación esperando cobrar vida. Ya sea una _startup_ o un proyecto consolidado, la elección de la herramienta de desarrollo adecuada es crucial para llevar esa idea desde su concepción hasta una realidad funcional y eficaz. En esta era digital, dos enfoques han capturado la atención: _low-code_ y _no-code_, destacando por su promesa de rapidez y accesibilidad para transformar ideas en aplicaciones plenamente operativas. Sin embargo, no podemos ignorar el pilar establecido de _Ruby on Rails_, un marco de desarrollo que ha resistido la prueba del tiempo y ha demostrado su eficacia en proyectos que requieren un rápido desarrollo. En esta comparación detallada, exploraremos cómo estas herramientas abordan el desafío de dar vida a las ideas, sopesando la agilidad de _low-code_ y _no-code_ con la robustez de Ruby on Rails. Descubriremos que, en última instancia, el éxito de cualquier idea radica en elegir la herramienta que fusiona eficiencia y efectividad para lograr soluciones duraderas. ¡Vamos a adentrarnos en esta fascinante comparativa!
+
+![Low-code y No-code ]({{page.images_path}}/low-code-no-code.png)
+## Empecemos, ¿qué c#$%!s son _low-code_ y _no-code_?
+
+Estos términos hacen referencia a herramientas con un entorno visual muy amigable y fácil de usar, diseñado para desarrollar y desplegar aplicaciones sin necesidad de conocimientos profundos en programación. El "low-code" implica que aunque se puede añadir código si es necesario, la mayoría de las veces no es esencial. Por otro lado, el "no-code" se centra en la creación de aplicaciones sin escribir una sola línea de código. En esencia, ambos enfoques están diseñados para que puedas tener tu aplicación funcional con solo unos cuantos clics, facilitando enormemente el proceso de materializar tu idea en un producto real y listo para el uso.
+
+### Ventajas de _low-code_ y _no-code_
+
+* **Rapidez de Desarrollo:** Agilidad en la creación de aplicaciones en comparación con métodos tradicionales.
```

Comment:
Esto es opinión no más, creo que los títulos de los ítems de un punteo no deberían ir en formato titular.
Si hay una regla que dice que después del `:` la letra sigue en minúscula, a menos que sea un nombre propio.

Cambiaría un poco el texto

```suggestion
* **Rapidez de desarrollo:** ágil en la creación de aplicaciones en comparación con métodos tradicionales.
```

Pero también te lo dudo, qué es lo ágil? Sales rápido con el `Hola mundo`, pero después de eso?
Yo sería bien explícito de dónde está esa supuesta velocidad.
Considera Visual Basic, o incluso WPF (si se sigue llamando así) Haces la interfaz con point and click en 1 minuto, pero no se si se les considera herramientas low-code.

Reply:
* Tienes toda la razón, debió ser un detalle que se me pasó en su momento.
* Con tradicional, hacía referencia a que debes inicializar un proyecto, configurar dependencias, definir una arquitectura y empezar a picar código. Pero tienes razón, esto debería ser mucho más claro.